### PR TITLE
get favorite vessels by user id API is added ;)

### PR DESCRIPTION
**Pull request by ghost**
You can get a list of favorite vessels by a User Id.
/api/users/user_id/vessels :GET
Auth: Super user, Admin, Operators

### CODE REVIEW

**Review by zkwsk — DISMISSED** [review]

**Comment by ghost** [comment]
sorry I pushed the wrong button.

File: src/server/api/routes/user.router.js [hunk 1]
```diff
@@ -3,15 +3,14 @@
 // router setup
 
 const express = require("express");
-const router = express.Router({
```
Comment (ismailirahim) [inline src/server/api/routes/user.router.js]: i think that your prettier settings are not right

File: src/server/api/routes/user.router.js [hunk 2]
```diff
@@ -51,9 +50,7 @@ router.post(
   authorizeUser(ROLES.SUPER_USER, ROLES.ADMIN),
   (req, res, next) => {
     userController
-      .createUser({
```
Comment (ismailirahim) [inline src/server/api/routes/user.router.js]: same here your prettier settings are not right

File: src/server/api/routes/user.router.js [hunk 3]
```diff
@@ -65,9 +62,7 @@ router.patch(
   authorizeUser(ROLES.SUPER_USER, ROLES.ADMIN, ROLES.OPERATOR),
   (req, res, next) => {
     userController
-      .editUser({
```
Comment (ismailirahim) [inline src/server/api/routes/user.router.js]: same here your prettier settings are not right

File: src/server/api/routes/user.router.js [hunk 4]
```diff
@@ -79,10 +74,7 @@ router.patch(
   authorizeUser(ROLES.SUPER_USER),
   (req, res, next) => {
     userController
-      .editUserRole({
```
Comment (ismailirahim) [inline src/server/api/routes/user.router.js]: same here your prettier settings are not right

File: src/server/api/routes/user.router.js [hunk 5]
```diff
@@ -91,29 +83,23 @@ router.patch(
 // ENDPOINT: /api/users/sendPasswordResetCode :PATCH
 router.post("/sendPasswordResetCode", (req, res, next) => {
   userController
-    .sendPasswordResetCode({
```
Comment (ismailirahim) [inline src/server/api/routes/user.router.js]: same here your prettier settings are not right

File: src/server/api/routes/user.router.js [hunk 6]
```diff
@@ -91,29 +83,23 @@ router.patch(
...
-    })
+    .sendPasswordResetCode({ body: req.body })
     .then(result => res.json(result))
     .catch(next);
 });
 
 // ENDPOINT: /api/users/resetPasswordWithCode :PATCH
 router.post("/resetPasswordWithCode", (req, res, next) => {
   userController
-    .resetPasswordWithCode({
```
Comment (ismailirahim) [inline src/server/api/routes/user.router.js]: same here your prettier settings are not right

File: src/server/api/routes/user.router.js [hunk 7]
```diff
@@ -91,29 +83,23 @@ router.patch(
...
     .then(result => res.json(result))
     .catch(next);
 });
 
 // ENDPOINT: /api/example/ :PATCH
 router.post("/changePasswordRandomly", (req, res, next) => {
   userController
-    .changePasswordRandomly({
-      body: req.body
-    })
```
Comment (ismailirahim) [inline src/server/api/routes/user.router.js]: same here your prettier settings are not right

File: src/server/api/routes/user.router.js [hunk 8]
```diff
@@ -124,9 +110,19 @@ router.delete(
   authorizeUser(ROLES.SUPER_USER, ROLES.ADMIN),
   (req, res, next) => {
     userController
-      .deleteUser({
-        body: req.body
-      })
```
Comment (ismailirahim) [inline src/server/api/routes/user.router.js]: same here your prettier settings are not right

**Review by ismailirahim — CHANGES_REQUESTED** [review]
you have to pull the latest version of develop in your branch.

**Comment by ghost** [comment]
@zkwsk Please review it and accept it because it is working and we test it last day.

**Review by zkwsk — APPROVED** [review]
Looks good!

**Pull request merged**